api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -402,6 +402,10 @@
      */
     Attribute<Boolean> BUFFER_SPOOLING = new Attribute<Boolean>(true);
     /**
+     * Whether to apply back pressure and stall the upstream operator when the downstream operator is slow.
+     */
+    Attribute<Boolean> BUFFER_BACK_PRESSURE = new Attribute<Boolean>(true);
```

Comment:
@PramodSSImmaneni It depends on an aggregation function. Aggregation does not always mean reduce. Independently, whether it is a simple accumulation or additional reduce, in my use case data may be accumulated in an operator for a long period of time leading to a large output size even after optional reduce.

In a DAG that reads data from Kafka, parses it, aggregates and continue processing, the back pressure needs to be enabled between Kafka and the aggregate operator but be disabled for the aggregate operator output.